### DISCUSSION TITLE
Who actually owns Scratch Addons?

### DISCUSSION THREAD

**Discussion by FunctionalMetatable** [ScratchAddons/ScratchAddons#1591] [Help] [answered]
Who does actually own scratch addons? is it @WorldLanguages  or @griffpatch ?

**Comment by jeffalo** [comment, marked as answer]
no one "owns scratch addons" it is a group effort.

world languages would probably be considered the leader, but both world languages and griffpatch have publishing rights to the chrome webstore.

griffpatch's contribution is mostly the original messaging extensions and the dev tools addon

**Reply by ghost** [reply]
But _who_ came up with the idea of Scratch Addons?

**Reply by FunctionalMetatable** [reply]
ohh ok

**Reply by mxmou** [reply]
> But _who_ came up with the idea of Scratch Addons?

World_Languages.

**Reply by WorldLanguages** [reply]
Worth noting that griffpatch can revoke my store publishing rights, but not the other way around.

**Reply by FunctionalMetatable** [reply]
*proves that griffpatch manages scratch* 
just a joke
XD

**Reply by WorldLanguages** [reply]
Oh, and I own the ScratchAddons.com domain (and of course, I'm the one paying the bill every year :P)

**Reply by FunctionalMetatable** [reply]
> Oh, and I own the ScratchAddons.com domain (and of course, I'm the one paying the bill every year :P)

Wish there was a company that gives free .com domains to open source projects

**Reply by Hans5958** [reply]
> 
> 
> Worth noting that griffpatch can revoke my store publishing rights, but not the other way around.

Why though? At least we can move FF to our AMO user, right?

**Reply by WorldLanguages** [reply]
@Hans5958 Not sure, that's how it is and I have no specific reason to ask that to change
